raster/src/main/scala/geotrellis/raster/summary/polygonal/visitors/MaxVisitor.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.raster.summary.polygonal.visitors
+
+import geotrellis.raster._
+import geotrellis.raster.summary.CellVisitor
+
+object MaxVisitor {
+  implicit def toTileVisitor(t: MaxVisitor.type): TileMaxVisitor =
+    new TileMaxVisitor
+  implicit def toMultibandTileVisitor(
+      t: MaxVisitor.type): MultibandTileMaxVisitor = new MultibandTileMaxVisitor
```

Comment:
Weird indentation is weird. Super-nit-picky but what is this intentional?

Reply:
Yeah this is weird. Not intentional. Must have missed one with scalafmt.